App/Config/Router.js

```diff
@@ -0,0 +1,101 @@
+import React, { Component } from 'react';
+
+import { StackNavigator, TabNavigator } from "react-navigation";
+
+import { FontAwesome } from "react-native-vector-icons";
+
+import SignIn from "../Screens/Main";
+import SignUp from "../Screens/SignUp";
+import Home from "../Screens/Search";
+import Account from "../Screens/Account";
+import Dashboard from "../Screens/Dashboard";
+import Profile from "../Screens/Profile";
+import Repositories from "../Screens/Repositories";
+import Notes from "../Screens/Notes";
+
+export const SignedOut = StackNavigator({
+  SignIn: {
+    screen: SignIn,
+    navigationOptions: {
+      title: "Sign In"
+    }
+  },
+  SignUp: {
+    screen: SignUp,
+    navigationOptions: {
+      title: "Sign Up"
+    }
+  }
+});
+
+export const SignedIn = TabNavigator({
+  Home: {
+    screen: Home,
+    navigationOptions: {
+      tabBarLabel: "Home",
+      //  tabBarIcon: ({ tintColor }) => (
```

Comment:
Merging commented out code into master is something you usually want to avoid, but I understand why these are here and I'm okay with it as long as ya clean it up at some point.